torch/ao/quantization/fx/convert.py

```diff
@@ -105,10 +104,14 @@ def _replace_observer_with_quantize_dequantize_node_decomposed(
 
     or quantize_per_channel and dequantize_per_channel
     """
+    graph = model.graph
     assert modules is not None
     assert isinstance(node.target, str)
     module_path, prefix = _get_module_path_and_prefix(node, node_name_to_scope, node_name_to_qconfig)
     activation_post_process = modules[node.target]
+    if hasattr(activation_post_process, "convert"):
+        activation_post_process.convert(model, node)
+        return
```

Comment:
I would suggest filing a todo where we refactor all observers with convert method.

Reply:
yeah will do